tests/services/test_search.py

```diff
@@ -196,86 +192,3 @@ def test_output_includes_box_number(self):
         )
         output_person: Person = output.scenarios[0].populations["employees"].people[0]
         assert output_person.box_number == "351234"
-
-
-class TestPersonOutputTranslator:
```

Comment:
This moved into its own file, and was mostly rewritten since a lot of its functionality was gutted.